pkg/restapi/operation/operations_test.go

```diff
@@ -232,21 +232,26 @@ func TestHandleOIDCCallback(t *testing.T) {
 	})
 
 	t.Run("generic bootstrap store FETCH error", func(t *testing.T) {
-		sub := uuid.New().String()
+		id := uuid.New().String()
```

Comment:
@llorllale I think these were missed in the last PR - let me know if you intended for these to be `sub` still.